middle_end/flambda/un_anf.ml

```diff
@@ -156,83 +156,100 @@ let make_var_info (clam : Clambda.ulambda) : var_info =
           ignore_debuginfo dbg;
           ignore_var_option env)
         functions
+        environment_vars
     | Uoffset (expr, offset) ->
       loop ~depth ~environment_vars expr;
-      ignore_int offset
+      ignore_int offset;
+      environment_vars      
     | Ulet (_let_kind, _value_kind, var, def, body) ->
       add_definition t (VP.var var) depth;
       loop ~depth ~environment_vars def;
```

Comment:
The compiler should complain about ignoring the result of this computation (and if it does not, I will :)).
This should look like `let environment_vars = loop ~depth ~environment_vars def in`.